py_zipkin/zipkin.py

```diff
@@ -709,3 +713,125 @@ def create_http_headers_for_new_span(context_stack=None, tracer=None):
         'X-B3-Flags': '0',
         'X-B3-Sampled': '1' if zipkin_attrs.is_sampled else '0',
     }
+
+
+def is_valid_trace_id(candidate):
+    return candidate and re.match(r'(?:[0-9a-f]{16}|[0-9a-f]{32})$', candidate)
+
+
+def is_valid_span_id(candidate):
+    return candidate and re.match(r'[0-9a-f]{16}$', candidate)
+
+
+def _extract_zipkin_attrs_from_single_header(b3_header, sample_rate,
+                                             use_128bit_trace_id):
+    # b3={TraceId}-{SpanId}-{SamplingState}-{ParentSpanId}
+    # where the last two fields are optional.
+    bits = b3_header.split('-')
+    if len(bits) == 1:
```

Comment:
Add a comment that there could also be just 1 field and that's the sampling decision. Otherwise this if seems in contradiction with the above comment that says only the last 2 fields are optional

Reply:
Done